spec/features/step_by_step_navigation_spec.rb

```diff
@@ -20,12 +23,71 @@
       then_I_visit_the_navigation_steps_page_again
       and_I_see_my_selected_preferences
     end
+
+    scenario "User schedules publishing" do
```

Comment:
These tests are really great, but given they are about publishing rather than setting navigation, perhaps they should live in `managing_step_by_step_pages_spec.rb` with the rest of the publishing tests?